deploy/post_deploy_testing/cypress/integration/10a_sayt_submission_view.js

```diff
@@ -0,0 +1,383 @@
+
+import { navUserAcctDropdownBtnSelector } from './../support/variables';
+
+describe('Search As You Type functionality on SubmissionView', function () {
+    var testItemsToDelete = [];
+    let principalObject; // temp id for main item (NOT db-generated @id); used for aliasing
```

Comment:
Would rename it to `newItemLocalAlias` or something to make upstream lines/references/usages of the variable more clear. `..Object` being present in variable name especially makes me initially/subconsciously think of it being an actual JS object until consciously remember/read/trace its context.